test/slim_check.lean

```diff
@@ -64,3 +65,98 @@ run_cmd do
 run_cmd do
   some p ← tactic.run_rand (find_prime 100000) | trace "no prime found, gave up",
   when (¬ nat.prime p) (trace!"The number {p} fooled Fermat's test")
+
```

Comment:
Could we move the three `run_cmd` tests above to the bottom of the file (or elsewhere)? For someone reading the tests as documentation, it's best if we start with the tests of the user facing behaviour of `slim_check`.